developer/web-services/index.md

```diff
@@ -10,26 +10,36 @@ categories:
+How can we be so sure?—Because we’re the experts in Utah geospatial data. 
 
-The searching API really opens up the opportunity for serious information gathering. The [SGID]({{ "/data/sgid-index/" | prepend: site.baseurl }}) has over 300 layers of real world data that you can run queries against.
+AGRC was developed specifically to coordinate Utah-specific data, and we’ve been doing this for quite a while (30+ years. But who’s counting?). Other programs can provide you with similar data, but if you’re looking for information that considers the nuances of Utah addresses, this is it—we’ve got the most comprehensive Utah geospatial data, and we want to share it with you. 
 
-#### Remember that database full of house addresses?
-You can learn so much from a house address once it has been geocoded. You could use our **search API** to find out how far away addresses are from a fault line, what school district the kids are in, what the broadband availability is and the list goes on. Head over to the [data]({{ "/data/" | prepend: site.baseurl }}) pages of our website and look at all the information you can extract to solve a problem or provide context.
+Our other main API, _**searching**_, allows you to (you guessed it!) search through SGID data. With over 300 layers of real-world data that you can run queries against in the SGID, the possibilities can be a little overwhelming. But the searching API really opens up the opportunity for serious information gathering. You need a way to sift through all that information and make sense of it, and our search API can help you do that.
 
-We haven't thought of all the great ideas you can build with our API but here are a few things we've already done to get your imagination going.
+# APIs in Action
+## Do you have a database full of house addresses?
+You can learn so much from a house address once it has been geocoded. When you geocode, you unlock the ability to visualize your data in a different way. You can use our search API to answer endless questions about your addresses: 
+  * How close is the nearest fault line?
+  * What school district will the kids be in?
+  * What is the broadband availability in my area?
+  * Where are the nearest public transit routes and stops?
+  * Where do I go to vote?
+  * Where is the closest post office?
 
- - [Find out who you Senator and Representative are by your address](http://le.utah.gov/GIS/findDistrict.jsp)
- - [Be transparent about environmental hazards with your customers](http://enviro.deq.utah.gov/)
- - [Aid in the decision making process to restore and manage ecosystems in priority areas across the state of Utah's Wildlife](http://wri.utah.gov/)
+This list could go on and on. In fact, we’ve barely scratched the surface of all the great ideas you can build with our API. Take a look at our [data pages](https://gis.utah.gov/data/) to get even more ideas about all the information you can extract to solve a problem or provide context. 
```

Comment:
API vs APIs. I think we need to be a bit more consistent with the naming. If we say API I think it should be referencing the Web API. If we say APIs then we should be talking about the geocoding, searching, or info endpoints. Maybe we shouldn't mix the two. API is the Web API and APIs are endpoints.